### PR TITLE
Add tkn to catalog.

### LINKED ISSUE #154: Add tkn task
We should add a task for calling [tkn](https://github.com/tektoncd/cli).

/assign wlynch

### PR DESCRIPTION
**Pull request by wlynch**
# Changes

Adds a Task to the catalog for the [Tekton CLI](https://github.com/tektoncd/cli).

Fixes #154

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)
- [x] Yaml file complies with [yamllint](https://github.com/adrienverge/yamllint) rules.

_See [the contribution guide](https://github.com/tektoncd/catalog/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: tkn/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+```
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/tkn/tkn.yaml
+```
+
+## Inputs
+
+### Parameters
+
+name      | description                                 | default
```
Comment (vdemeester) [inline tkn/README.md]: Should we do the same for the rest ? (We need templates and a small script like `./create-task` that would create the folder and put some skeleton in there)

File: tkn/tkn.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+spec:
+  inputs:
+    params:
+      - name: tkn-image
+        description: tkn CLI container image to run this task
+        default: gcr.io/tekton-releases/dogfooding/tkn
+      - name: ARGS
+        type: array
+        description: tkn CLI arguments to run
+        default: ["help"]
```
Comment (vdemeester) [inline tkn/tkn.yaml]: Should we have no `default` for the task, thus making the `ARGS` a required argument.

File: tkn/tkn.yaml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: tkn
+spec:
+  inputs:
+    params:
+      - name: tkn-image
+        description: tkn CLI container image to run this task
+        default: gcr.io/tekton-releases/dogfooding/tkn
```
Comment (danielhelfand) [inline tkn/tkn.yaml]: We should tag a stable release for the default.

File: tkn/tkn.yaml (hunk 2 above)
Comment (wlynch) [reply tkn/tkn.yaml]: Removed.

File: tkn/tkn.yaml (hunk 3 above)
Comment (wlynch) [reply tkn/tkn.yaml]: AFAIK this is the best we have right now. Once https://github.com/tektoncd/cli/issues/497 is resolved, we can update this URL.

Comment (danielhelfand) [reply tkn/tkn.yaml]: Yes, my apologies. I opened that issue after seeing this pr and became aware from it that we need an official image.

**Pull request merged**